### ISSUE TITLE
Adding more recent techniques e.g. WeightInit

### ISSUE THREAD

**Issue by ZhongBaby** [deeplearning4j/deeplearning4j#2590]
I'd like some more recent techniques like [WeightInit in keras](https://keras.io/initializations/).

Because I encounter difficulty in training the [same architecture](http://ankivil.com/kaggle-first-steps-with-julia-chars74k-first-place-using-convolutional-neural-networks/) which has 13 layers and achieve 86% accuracy in Kaggle's google street image character classification.

Except dl4j lack the he_normal initialization.

**Comment by AlexDBlack** [comment]
Read the javadoc here:
https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/weights/WeightInit.java

RELU and RELU_UNIFORM correspond to Keras' he_normal and he_uniform respectively.
Glorot -> Xavier
Issue for orthogonal open here: https://github.com/deeplearning4j/deeplearning4j/issues/2260

**Comment by ZhongBaby** [comment]
I definitely need to read more before asking!:(

**Comment by AlexDBlack** [comment]
That's fine. If you can't find something in our documentation, feel free to ask us on gitter: https://gitter.im/deeplearning4j/deeplearning4j
